### PR TITLE
Added --page-all option to download all search results

### PR DESCRIPTION
**Pull request by Nontre12**
`nhentai --search "darling in the franxx" --page-all`  

Output example:

```
[18:44:03] [INFO] Search Result || Found 108 doujinshis 
======  =======================================================================================
    id  doujinshi
======  =======================================================================================
328231  (C94) [Workaholic (Kni)] Robo Sukebe Bon | Robo Hentai Book [English] [Fated Circle]
318600  [Carrot Academy (carrot)] 満心総意の躾 (DARLING in the FRANXX)
313683  (C94) [Workaholic (Kni)] robo-hentai-book
308637  (GOOD COMIC CITY 25) [Bad Mushrooms (4why&Chicke III, mimo)] DARLING Ningen Shikkaku ..
300967  DArling Baby Maker
294623  zero tsu doujin

[summary (lots of doujinshis)]

228567  [Kouchaya (Ootsuka Kotora)] GERBERA (DARLING in the FRANXX) [Chinese] [oo君個人漢化] [Digi..
228527  [Kouchaya (Ootsuka Kotora)] GERBERA (DARLING in the FRANXX) [Digital]
228291  [Hirani (Ginhaha, HiRA'n)] Darling need more Sexx (DARLING in the FRANXX) [English] [..
227901  [Taiyaki] Ichigo to Goro no Ero Manga (DARLING in the FRANXX)
226970  [Hirani (Ginhaha, HiRA'n)] Darling need more Sexx (DARLING in the FRANXX) [English] [..
226416  [hrd] Goro no Sainan na Yoru (DARLING in the FRANXX) [English]
225882  [hrd] Goro no Sainan na Yoru (DARLING in the FRANXX)
225556  [Hirani (Ginhaha, HiRA'n)] Darling need more Sexx (DARLING in the FRANXX) [Chinese] [..
======  =======================================================================================
[18:44:03] [INFO] All done.

```

### CODE REVIEW

File: nhentai/parser.py [hunk 1]
```diff
@@ -247,6 +247,38 @@ def search_parser(keyword, sorting, page):
     return result
 
 
+def search_parser_all(keyword):
```
Comment (RicterZ) [inline nhentai/parser.py:230]: I think the function can merge to function search_parser.

Comment (Nontre12) [reply nhentai/parser.py:230]: Without, **'page'** parametter?

Comment (RicterZ) [reply nhentai/parser.py:230]: I'm working on it, I'll metion you after pushing

**Review by Nontre12 — COMMENTED** [review]
On `search_parser_all(keyword)` function, needs replace **print() lines** by **logger.py** file functions.

**Pull request merged**